snuba/datasets/configuration/transactions/entities/transactions.yaml

```diff
@@ -347,3 +347,6 @@ validators:
   ]
 
 required_time_column: finish_ts
+subscriptions:
```

Comment:
How is `entity_subscription=None` supposed to be represented here? Does the absence of `subscriptions` mean none?

Also, please align all the names in the yaml and python versions. In python you called it `entity_subscription`. Here the key is `subscriptions`. It is much clearer for the user/reader if all of these names which essentially represent the same thing line up

Reply:
Yes, the absence of `subscription` means there is no entity_subscription set which means this entity does not use subscriptions. This idea of "define what you need" is true for the way we define datasets as config right now. For example, a user can leave out `writeable_storage` which implies that `writeable_storage=None`

Sure, I can update the name to `entity_subscription` align with python definition.